.github/workflows/packer-build-and-publish.yml

```diff
@@ -215,6 +274,19 @@ jobs:
           git tag -d ${{ github.ref_name }}
           git tag -a ${{ github.ref_name }} -m "Release ${{ github.ref_name }}"
           git push -f origin ${{ github.ref_name }}
```

Comment:
Can I convince you that we should rework how this tag/force-push works? To me, there are a couple of other ways we could do this:

  - Have a `Makefile` target or script to update the READMEs locally, then commit and push the tag. In this flow, we could even require signed tags.
  - Have a `workflow_dispatch` action which does the same.
  - Use GitHub releases - created in response to a tag event & after the image is built - and store the README contents in the release notes there.

(Try not to get too hung up on these specific suggestions in case any of them won't work, but instead think about the intention behind them.)

The tag should be thought of as the developer's stamp on a particular commit. It says "this is the state that I want to release". By overwriting it we remove that piece of information, we change their intention (what is released is a bit different to what they had in their hands), and we lose that piece of traceability.

Reply:
The READMEs are updated by the Software Report step during packer image generation.